packages/sources/genesis-volatility/src/config.ts

```diff
@@ -0,0 +1,12 @@
+import { Requester } from '@chainlink/ea-bootstrap'
+import { Config } from '@chainlink/types'
+
+export const NAME = 'GENESIS_VOLATILITY'
+
+export const DEFAULT_BASE_URL = 'https://app.pinkswantrading.com/graphql'
+
+export const makeConfig = (prefix?: string): Config => {
+  const config = Requester.getDefaultConfig(prefix, true)
+  config.api.baseURL = config.api.baseURL || DEFAULT_BASE_URL
```

Comment:
I believe every instance of this line will need to be 
```
  config.api.baseUrl = config.api.baseURL || DEFAULT_BASE_URL
```

to be able to be spread into Axios' options.

Reply:
What's the difference here? All I see is baseUrl instead of baseURL but I'm not sure if that's intentional

Reply:
Just double checked, I was wrong. It should be `baseURL`: https://github.com/axios/axios#request-config.

Resolving this.